### DISCUSSION TITLE
Reduced scattering coefficient

### DISCUSSION THREAD

**Discussion by ABoamfa** [ankrh/MCmatlab#5] [Q&A] [answered]
Hello,

I wanted to ask if there is any manner to set the reduced scattering coefficient in the mediaProperties function.
From the tissue layer that I want to simulate, I only know the reduced scattering coefficient, but not the exact anisotropy or the scattering coefficient. I could make an estimated guess but I would prefer not to.
Thanks in advance for any help!

**Comment by ankrh** [comment, marked as answer]
Hi! Thanks for your interest in MCmatlab.

Unfortunately, the reduced scattering coefficient is not enough by itself to know how the photons propagate in the media. See for example the explanation at https://omlc.org/classroom/ece532/class3/musp.html#:~:text=The%20reduced%20scattering%20coefficient%20is,each%20step%20involves%20isotropic%20scattering.

My recommendation is to make a guess of what the scattering anisotropy g is based on its value for other, similar, media and then calculate the corresponding value of µ_s based on µ_s' = µ_s*(1 - g). Fortunately, the exact value used for g often doesn't matter that much for most simulations as long as you have the correct value of the reduced scattering coefficient µ_s'.

Try with a few different values of g, recalculating µ_s for each one, and see how much of a difference it makes for your particular model.

**Comment by ABoamfa** [comment]
Thank you a lot @ankrh for your answer! Then I will use a representative g and work from there.
